src/Core/TileMesh.js

```diff
@@ -37,6 +39,7 @@ function TileMesh(geometry, params) {
     this.frustumCulled = false;
 
     this.updateGeometricError();
+    this.wmtsCoords = {};
```

Comment:
I think it makes more sense to move this to `Extent`

Reply:
Yeah... I moved it here, again because it was assigned somewhere else (https://github.com/iTowns/itowns/pull/792/files#diff-5e0253bd7132719c7fe2322c297efe16L48).

I propose to keep it that way for this PR, and use your PR #785  as an opportunity to fix this since it's more focused on the extent side. WDYT?

Reply:
ok